app/modules/users/models.py

```diff
@@ -28,6 +31,15 @@
 PST = pytz.timezone('US/Pacific')
 
 
+class TimestampViewed(Timestamp):
```

Comment:
@brmscheiner I've added a new `TimestampViewed` class that inherits from the previous `Timestamp` on the `User` object.  The class creates a `viewed` attribute in the database for all users (and any object that uses the new viewed time stamp parent class) that records the most recent time the method `user.view()` is called.  We can add this method to all frontend, backend, and API endpoint handlers for Houston so that this value is updated automatically.